subprojects/diagnostics/src/main/java/org/gradle/api/tasks/diagnostics/internal/configurations/model/ReportConfiguration.java

```diff
@@ -30,13 +31,16 @@ public final class ReportConfiguration {
     private final String description;
     @Nullable
     private final Type type;
+
+    private List<? extends GradleException> lenientErrors;
```

Comment:
❗ Should be `final` and defensively copied to an immutable list, to keep this model immutable.